### ISSUE TITLE
What to do with images?

### ISSUE THREAD

**Issue by dufferzafar** [jlfwong/messenger-to-markdown#5]
In case an image is sent as a message, the markdown has no mention of it.

The least we could do is add a line say "*image* was sent."

**Comment by kwikadi** [comment]
Where is the line drawn, though? There are also nicknames, stickers, emoji, reactions, etc. I'm guessing most of these won't be supported either.

**Comment by dufferzafar** [comment]
Yeah, right, but then stickers/emoji don't effect the conversation much, right? IDK.

**Comment by dufferzafar** [comment]
The big FB like shows up in markdown as "Thumbs Up Sign", which is cool.

~~I wonder if other emojis will show up like this too.~~ No, they don't.

**Comment by jlfwong** [comment]
I'm happy to accept PRs for any of the things you listed 😄 

I'm unsure if the image URLs in messenger conversations are persistent or not (e.g. here's one for a recent conversation I had: https://scontent-sjc3-1.xx.fbcdn.net/v/t34.0-0/p280x280/30184605_10210321502947942_1592876221_n.png?_nc_cat=0&oh=badb3b4c393f389db2f1e5dc5fc2ad08&oe=5AD3FFF8). If those *are* persistent long-term, then it might suffice to just embed them via markdown images.

**Comment by jlfwong** [comment]
Also, I'm curious! What have you been using this for? I use this to archive bits of messenger conversations in my personal journal

**Comment by dufferzafar** [comment]
Hah!

Today, I used it to export chats I have with friends to share them with *other* friends.

I and @kwikadi wanted to write a blog post but most of our "research" was scattered around in chats with friends, so we exported it to a text editor, and went through it line by line. [The post in question.](https://medium.com/@meninthemiddle/the-three-assignments-fc32455c9ce5)
